v2.1/architecture/transaction-layer.md

```diff
@@ -84,7 +84,7 @@ For more detail about the risks that large clock offsets can cause, see [What ha
 
 To provide serializability, whenever an operation reads a value, we store the operation's timestamp in a timestamp cache, which shows the high-water mark for values being read.
 
-Whenever a write occurs, its timestamp is checked against the timestamp cache. If the timestamp is less than the timestamp cache's latest value, we attempt to move the timestamp for its transaction forward to a later time. In the case of serializable transactions, this causes them to restart in the second phase of the transaction.
+Whenever a write occurs, its timestamp is checked against the timestamp cache. If the timestamp is less than the timestamp cache's latest value, we attempt to push the timestamp for its transaction forward to a later time. In the case of serializable transactions, this might cause them to restart in the second phase of the transaction (see [read refreshing](#read-refreshing)).
```

Comment:
Doh. This is a place that we should update to clarify that all transactions are serializable. Do you mind if I piggy-back the edits on this PR, or you want to merge this and I'll open a new one?

Reply:
Feel free to piggy-back / take this one over.  Thanks!